daily_read/order_portal.py

```diff
@@ -14,7 +15,7 @@
 class OrderPortal(object):
     """Class to handle NGI order portal interaction"""
 
-    def __init__(self, config_values):
+    def __init__(self, config_values, projects_data):
         self.config_values = config_values
         base_url = config_values.ORDER_PORTAL_URL
```

Comment:
I had an issue with urljoin removing the `/orders` part of my url `https://ngisweden.scilifelab.se/orders`. I had to add the trailing `/` for it to be kept. We should probably check that the base_url has a trailing `/`...

Reply:
I made an issue for later: https://github.com/NationalGenomicsInfrastructure/DailyRead/issues/17